apps/redi-talent-pool/src/pages/app/browse/BrowseCompany.tsx

```diff
@@ -33,12 +35,23 @@ export function BrowseCompany() {
   const { name, skills, desiredPositions, isJobFair2022Participant } = query
 
   const history = useHistory()
+
   const { data: jobseekerProfiles } = useBrowseTpJobseekerProfilesQuery({
     name,
     skills,
     desiredPositions,
     isJobFair2022Participant,
   })
+  const { data: companyProfile } = useTpCompanyProfileQuery()
+  const tpCompanyProfileUpdateMutation = useTpCompanyProfileUpdateMutation()
+
+  const toggleFavorites = (favoritesArr, value) => {
```

Comment:
Perhaps redundant to pass in `favoritesArr` here as it's easily accessible via the closure?